component/component/provider.jsonnet

```diff
@@ -173,8 +173,8 @@ local runtimeConfigRef(name) = {
         },
         {
           apiGroups: [ 'rbac.authorization.k8s.io' ],
-          resources: [ 'clusterrolebindings', 'roles', 'rolebindings' ],
-          verbs: [ 'get', 'list', 'watch', 'update', 'patch', 'create', 'delete' ],
+          resources: [ 'clusterrolebindings', 'roles', 'rolebindings', 'clusterroles' ],
+          verbs: [ '*' ],
```

Comment:
Please don't use `'*'` for verbs. We'll get flagged by automated security scanners with this.